### ISSUE TITLE
Request: Update instructions to reflect Sage 5.2

### ISSUE THREAD

**Issue by NicholasKirchner** [sagemath/sagecell#352]
I've been trying to install sagecell starting from a freshly compiled sage-5.2rc1.  First: installing the sagecell package immediately did not work.

Second, I read the darn instructions and applied patches from 12719 and 13121 tracs.  Applying the patches seemed to work OK.  Still, sagecell would not install.  The error message I was getting was `Error: you have an out-of-date 02-sage-show.patch`.  Google led me to the link https://groups.google.com/forum/#!topic/sage-gsoc/P7xozeAcguY

Third: attempting the command (from the path SAGE_ROOT/devel/sage) `../../sage -hg qpop 02-sage-show.patch`
resulted in the error message `about: patch 02-sage-show.patch not in series`.  `../../sage -hg qseries` did not list 02-sage-show.patch, but it did list 01-sage-embedded.patch.  I deleted that patch, and tried to install sagecell again.  It failed with more error messages (mostly about patches not being applied properly).

In short, all the instructions I managed to find for earlier Sage versions seemed to fail one way or another.
